web/src/containers/activity/Goals.js

```diff
@@ -1,61 +1,133 @@
+import { Button, Review, TextField } from '@cmsgov/design-system-core';
+
 import PropTypes from 'prop-types';
-import React, { Component, Fragment } from 'react';
+import React, { Component, Fragment, useState } from 'react';
 import { connect } from 'react-redux';
 
 import {
   addActivityGoal as addActivityGoalAction,
   removeActivityGoal as removeActivityGoalAction,
   updateActivity as updateActivityAction
 } from '../../actions/activities';
-import Btn from '../../components/Btn';
-import CollapsibleList from '../../components/CollapsibleList';
-import { RichText } from '../../components/Inputs';
-import Instruction from '../../components/Instruction';
 import NoDataMsg from '../../components/NoDataMsg';
 import { Subsection } from '../../components/Section';
 import { t } from '../../i18n';
 
-const plaintext = txt => {
-  const e = document.createElement('div');
-  e.innerHTML = txt;
-  return e.innerText;
+const GoalReview = ({ goal, idx, edit, handleDelete }) => (
+  <Review
+    heading={
+      <h5 className="ds-h4">
```

Comment:
@hbillings Should this be `<h4 className="ds-h4">`?  The header above is an `h3` and the React nesting validator is complaining about having a 5 directly under a 3.  I wasn't clear on why it was a 5 so I didn't change it yet, but raising because...  I think maybe it should be a 4.  😛

Reply:
The heading levels are my best attempt at guessing what things should be, which is super hard to do piecemeal like this. If the validator is complaining, it knows better than I do!

The one thing that seems like it might throw a wrench into this is where reviews are used in nested elements (activities). Those probably need to have a heading level that is lesser than the heading levels of non-nested components, so maybe we make the level/class overrideable but with sensible defaults? (I did something like this in the cost allocation PR with the `Instruction` component: https://github.com/18F/cms-hitech-apd/pull/1569/files#diff-1f5a9d3d843586bd0de4807c9b3f7442R7)